addons/point_of_sale/security/ir.model.access.csv

```diff
@@ -15,24 +15,12 @@ access_account_payment_method_line_pos_user,account.payment.method.line pos_user
 access_account_bank_statement_line,account.bank.statement.line,account.model_account_bank_statement_line,group_pos_user,1,1,1,0
 access_product_product,product.product,product.model_product_product,group_pos_user,1,0,0,0
 access_product_template_pos_user,product.template pos user,product.model_product_template,group_pos_user,1,0,0,0
-access_product_product_pos_manager,product.product.pos manager,product.model_product_product,group_pos_manager,1,1,1,1
-access_product_template_pos_manager,product.template pos manager,product.model_product_template,group_pos_manager,1,1,1,1
-access_product_attribute_pos_manager,product.attribute,product.model_product_attribute,group_pos_manager,1,1,1,1
-access_product_attribute_value_pos_manager,product.attribute value,product.model_product_attribute_value,group_pos_manager,1,1,1,1
-access_update_product_attribute_value_pos_manager,update.product.attribute.value,product.model_update_product_attribute_value,group_pos_manager,1,1,1,0
-access_product_attribute_custom_value_pos_manager,product.attribute.custom value manager,product.model_product_attribute_custom_value,group_pos_manager,1,1,1,1
-access_product_product_attribute_pos_manager,product.template.attribute value,product.model_product_template_attribute_value,group_pos_manager,1,1,1,1
-access_product_template_attribute_exclusion_pos_manager,product.template.attribute exclusion,product.model_product_template_attribute_exclusion,group_pos_manager,1,1,1,1
-access_product_template_attribute_line_pos_manager,product.template.attribute line,product.model_product_template_attribute_line,group_pos_manager,1,1,1,1
 access_account_bank_statement_line_manager,account.bank.statement.line manager,account.model_account_bank_statement_line,group_pos_manager,1,1,1,1
 access_product_product_supplierinfo_user,product.supplierinfo user,product.model_product_supplierinfo,group_pos_user,1,0,0,0
 access_stock_warehouse_manager,stock.warehouse manager,stock.model_stock_warehouse,group_pos_manager,1,0,0,0
 access_stock_location_manager,stock.location manager,stock.model_stock_location,group_pos_manager,1,0,0,0
-access_uom_uom_manager,uom.uom manager,uom.model_uom_uom,group_pos_manager,1,1,1,1

```

Comment:
Why the uom access removal?
Doesn't seem related to the product administrator group :thinking: